README.md

```diff
@@ -54,7 +53,14 @@ require("zk").setup({
       enabled = true,
       filetypes = { "markdown" },
     },
+
+    -- can be "telescope", "fzf" or "select" (`vim.ui.select`)
+    picker = "telescope",
```

Comment:
I’m not entirely sure, but it might be wiser to default to what everyone has installed, i.e. `vim.ui.select`.

Reply:
Or we can detect if telescope is installed and if not fallback to `vim.ui.select`.

Reply:
Agreed

Reply:
At this point might as well also look for fzf?

Reply:
That’s a good point, I guess the default setting could try to guess which picker is install and fallback to `vim.ui.select`.

My only concern would be that it might significantly slow down nvim startup time, because we would be exploring the various paths multiple times (I realize there is some caching here, but still). That would be a reason to default to `vim.ui.select`, so that there is slowdown due to looking up for many things.

Reply:
I'll measure if there is any slow down probing for telescope > fzf > vim.ui.select.